### DISCUSSION TITLE
how to enable transparency and blur effect on win11

### DISCUSSION THREAD

**Discussion by ghost** [Simnico99/MicaWPF#103] [General]
![Sleekshot 2024-08-02 13-00-48](https://github.com/user-attachments/assets/cee9dbb8-32bb-42d8-95ba-dbb32dfd8df9)
the window doesn't have such beautiful effects of transparency, color blur, etc., no matter what I do

**Comment by Simnico99** [comment]
In Windows setting your transparency is off I can see from your taskbar being opaque
![image](https://github.com/user-attachments/assets/266bdca6-d236-4549-bf66-885ede1b99c5)
